pygeo/parameterization/DVGeoAxi.py

```diff
@@ -240,11 +240,11 @@ def addPointSet(self, points, ptName, origConfig=True, **kwargs):
 
         xform = self.axiTransforms[ptName] = _AxiTransform(points, self.center, self.collapse_into, self.complex)
 
-        super(DVGeometryAxi, self).addPointSet(xform.c_pts, ptName, origConfig, **kwargs)
+        super().addPointSet(xform.c_pts, ptName, origConfig, **kwargs)
```

Comment:
I am not very familiar with the use of `super()`, why were the previous arguments unnecessary?

Reply:
It's an unnecessarily verbose approach (which was mandatory in Python 2 I think). If a class only has one parent, `super()` will refer to it, there is no need to use `super(<class>, self)`